addons/im_livechat/static/tests/embed/feedback_panel.test.js

```diff
@@ -133,12 +133,17 @@ test("Start new session from feedback panel", async () => {
     await contains(".o-mail-Thread:not([data-transient])");
     await click("[title*='Close Chat Window']");
     await click(".o-livechat-CloseConfirmation-leave");
+    pyEnv["im_livechat.channel"].write([channelId], {
+        user_ids: [Command.clear(serverState.userId)],
+    });
     pyEnv["im_livechat.channel"].write([channelId], {
         user_ids: [
-            Command.clear(serverState.userId),
-            Command.create({ partner_id: pyEnv["res.partner"].create({ name: "Bob Operator" }) }),
```

Comment:
Note, mock server fails to assign "livechat_channel_ids" (new field) inverse with this syntax.